pkg/channel/distributed/common/constants/constants.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2021 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package constants
+
+// Global Constants
+const (
+	// Values Used When Initializing Cloud Events
+	DefaultMaxIdleConns        = 1000
+	DefaultMaxIdleConnsPerHost = 100
+)
```

Comment:
We have similar already here:

https://github.com/knative-sandbox/eventing-kafka/blob/main/pkg/channel/distributed/dispatcher/constants/constants.go#L22-L23

Folks, would it be possible to move all these constants to just one common file, to have em in a single place?

Reply:
the above is just one example - there are more occurrences in our codebase

Reply:
Certainly; I've moved them to pkg/common/constants/constants.go.  It should at least be easier to move them elsewhere if desired from there.